app/mailers/report_mailer.rb

```diff
@@ -1,30 +1,32 @@
 class ReportMailer < ActionMailer::Base
-  default :from => AppConfig.mail.sender_address
+  default from: AppConfig.mail.sender_address
 
-  def new_report(type, id)
+  def self.new_report(type, id)
+    [Role.admins + Role.moderators].flatten.map {|role| super(type, id, role) }
+  end
+
+  def new_report(type, id, role)
```

Comment:
How about using keyword arguments here to improve the readability of the method signature?